### PR TITLE
Adding flushtime to put

### PR DESCRIPTION
**Pull request by finmaguire**

### CODE REVIEW

File: pg2kinesis/__main__.py [hunk 1]
```diff
@@ -68,6 +72,7 @@ def __init__(self, formatter, writer):
         self.msg_window_size = 0
         self.msg_window_count = 0
         self.cur_window = 0
+        self.last_flush = datetime.now()
```
Comment (wai-kuen-shopkeep) [inline pg2kinesis/__main__.py]: How is `self.last_flush` being used in the code?

Comment (finmaguire) [reply pg2kinesis/__main__.py]: Was used to force a flush every 30 seconds but as that is not our approach now I have removed it. 
Good catch. :)

File: pg2kinesis/stream.py [hunk 2]
```diff
@@ -41,7 +41,7 @@ def put_message(self, fmt_msg):
             self._send_agg_record(agg_record)
             self.last_send = time.time()
 
-        return agg_record
+        return agg_record or True
```
Comment (wai-kuen-shopkeep) [inline pg2kinesis/stream.py]: This change forces `put_message()` to return true all the time.   In __main__.py, this means we flus the log every batch.   Is it the correct behaviour?

Comment (finmaguire) [reply pg2kinesis/stream.py]: again part of previous strategy we are no longer using. :)
removed

**Pull request closed without merging**